resources/teamcity/windows/windows-actions.inc.sh

```diff
@@ -0,0 +1,22 @@
+# Keyman is copyright (C) SIL Global. MIT License.
+
+windows_build_action() {
+  builder_echo start "build windows" "Building Keyman for Windows"
+  # shellcheck disable=SC2154
+  "${KEYMAN_ROOT}/windows/build.sh" configure build
+  builder_echo end "build windows" success "Finished building Keyman for Windows"
+}
+
+windows_test_action() {
+  builder_echo start "test windows" "Running Keyman for Windows tests"
+  "${KEYMAN_ROOT}/windows/build.sh" test
+  builder_echo end "test windows" success "Finished running Keyman for Windows tests"
+}
+
+call_windows_publish() {
```

Comment:
```suggestion
windows_call_publish() {
```

Suggesting using `windows_` as common prefix for windows-actions-inc.sh.

I wonder if this is even necessary as it's only a simple one-liner anyway. It might be better to just call the script directly from the callers?

Reply:
done